save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/DeleteOrganizationButton.kt

```diff
@@ -0,0 +1,82 @@
+package com.saveourtool.save.frontend.components.views.usersettings
+
+import com.saveourtool.save.frontend.components.modal.displayModal
+import com.saveourtool.save.frontend.utils.*
+import com.saveourtool.save.frontend.utils.noopLoadingHandler
+import csstype.ClassName
+import react.ChildrenBuilder
+import react.FC
+import react.PropsWithChildren
+import react.dom.html.ReactHTML.button
+import react.dom.html.ReactHTML.div
+
+/**
+ * DeleteOrganizationButton props
+ */
+external interface DeleteOrganizationButton : PropsWithChildren {
+    /**
+     * All filters in one class property [organizationName]
+     */
+    var organizationName: String
+
+    /**
+     * lambda to change [organizationName]
+     */
+    var onDeletionSuccess: () -> Unit
+
+    /**
+     * Button View
+     */
+    var buttonStyleBuilder: (ChildrenBuilder) -> Unit
+
+    /**
+     * classname for the button
+     */
+    var classes: String
+}
+
+/**
+ * Button for delete organization
+ *
+ * @return noting
+ */
+fun deleteOrganizationButton() = FC<DeleteOrganizationButton> { props ->
+    val windowOpenness = useWindowOpenness()
+
+    val deleteOrganization = useDeferredRequest {
+        val responseFromDeleteOrganization =
+                delete(
+                    "$apiUrl/organizations/${props.organizationName}/delete",
+                    jsonHeaders,
+                    body = undefined,
+                    loadingHandler = ::noopLoadingHandler,
+                )
+        if (responseFromDeleteOrganization.ok) {
+            props.onDeletionSuccess()
+        }
+    }
+
+    displayModal(
```

Comment:
can we use `displayConfirmationModal` instead of `displayModal` here?

Reply:
No, this is done in order to give out more text information in the buttons, paint them in more suitable colors

Reply:
ok, then.
also, need to close window after `deleteOrganization`: